reference/uitpas.json

```diff
@@ -2863,6 +2866,118 @@
         ],
         "description": "Retrieve information related to a chip number.\n\nIf the response contains `messages`, they MUST be displayed to the end-user.\n\nThe caller of this request must have `CHIP_NUMBERS_READ` permission."
       }
+    },
+    "/card-systems": {
+      "get": {
+        "summary": "Search card systems",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CardSystemsPaginatedCollection"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "totalItems": 0,
+                      "member": [
+                        {
+                          "id": 0,
+                          "name": "string",
+                          "branding": {
+                            "logo": "string",
+                            "primaryColor": "string",
+                            "secondaryColor": "string"
+                          },
+                          "links": {
+                            "website": "string",
+                            "terms": "string"
+                          },
+                          "cities": [
+                            {
+                              "postalCode": "string",
+                              "name": "string"
+                            }
+                          ],
+                          "permanent": true
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request.Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/invalid-city\n* https://api.publiq.be/probs/uitpas/invalid-postal-code\n\nThe detail property might include more information for the client developer."
```

Comment:
> * https://api.publiq.be/probs/uitpas/invalid-city
> * https://api.publiq.be/probs/uitpas/invalid-postal-code

If these query parameters can return these specific errors, it would be valuable info to document in the query parameters how they are validated.

Reply:
One is no longer relevant, for the other see my other comment